mxd/backend-service/src/main/java/org/eclipse/tractusx/mxd/backendservice/BackendServiceExtension.java

```diff
@@ -78,10 +78,8 @@ public class BackendServiceExtension implements ServiceExtension {
     @Inject
     private TypeManager typeManager;
 
-    @Inject(required = false)
     private ContentStatementsService statements;
```

Comment:
This is an internal class, so it should not be injected. Just remove this field. Only external dependencies should be injected.

Reply:
This injection has been removed already. It's red marked.